lib/tasks/canonical_models/canonical_misc_items.json

```diff
@@ -1387,39 +1387,7 @@
       ],
       "description": null,
       "purchasable": false,
-      "unique_item": true,
-      "rare_item": true,
-      "quest_item": true,
-      "quest_reward": false
-    }
-  },
-  {
```

Comment:
I had to delete two duplicates of the Opaque Vessel and the Werewolf Totem, since they were differentiable only by item code, and set the third duplicate to `unique_item: false`. Having the three copies of these models would've prevented a canonical model from ever being set on an in-game item with any of these as the canonical.